docs/accessibility.md

```diff
@@ -46,6 +46,23 @@ To use, set the `accessibilityLabel` property to a custom string on your View, T
 
 In the above example, the `accessibilityLabel` on the TouchableOpacity element would default to "Press me!". The label is constructed by concatenating all Text node children separated by spaces.
 
+### `accessibilityLabelledBy`
+
+A reference to another element nativeID used to build complex forms.
+The value of the `accessibilityLabelledBy` should match the `nativeID` of the other element:
```

Comment:
Small nits.

```suggestion
The value of `accessibilityLabelledBy` should match the `nativeID` of the related element:
```

Reply:
@huntie Thanks, Alex. I added the changes with commit https://github.com/facebook/react-native-website/commit/487a9259914b848c0f492a6f31c09a7d2d358f06